socorro/external/es/crashstorage.py

```diff
@@ -168,13 +184,39 @@ def reconstitute_datetimes(processed_crash):
                 # not there? we don't care
                 pass
 
+    @staticmethod
+    def remove_bad_keys(raw_crash):
```

Comment:
I appreciate that we only run this static method for the raw crash (because processed_crash is controlled any won't get crazy) but perhaps not call it `raw_crash` here because this, in fact, works for any mapping.